test/quantization/pt2e/test_quantize_pt2e_qat.py

```diff
@@ -159,8 +159,8 @@ def _verify_symmetric_xnnpack_qat_numerics_helper(
         self.assertEqual(after_prepare_result_pt2e, after_prepare_result_fx)
 
         if verify_convert:
-            torch.ao.quantization.move_exported_model_to_eval(model_pt2e)
             model_pt2e = convert_pt2e(model_pt2e)
+            torch.ao.quantization.move_exported_model_to_eval(model_pt2e)
```

Comment:
Actually please revert this part of the change. This diff looks different from the master version?

Reply:
Sorry for the mistake. I applied the patch before our discussion. The changes in this PR should be same as https://github.com/pytorch/pytorch/pull/114547 now. Please help to take a look again.